doc/source/quick_start.md

```diff
@@ -0,0 +1,262 @@
+---
+file_format: mystnb
+kernelspec:
+  name: geoutils
+---
+(quick-start)=
+
+# Quick start
+
+The following presents a descriptive example show-casing all core aspects of GeoUtils.
+
+For more details, refer to the {ref}`core-index`, {ref}`rasters-index` or {ref}`vectors-index` pages.
+
+To find an example about a specific functionality, jump to {ref}`quick-gallery`.
+
+## The core {class}`~geoutils.Raster` and {class}`~geoutils.Vector` classes
+
+In GeoUtils, geospatial handling is object-based and revolves around {class}`~geoutils.Raster` and {class}`~geoutils.Vector`.
+These link to either **on-disk** or **in-memory** datasets, opened by instantiating the class.
+
+```{code-cell} ipython3
+import geoutils as gu
+
+# Examples files: infrared band of Landsat and glacier outlines
+filename_rast = gu.examples.get_path("everest_landsat_b4")
+filename_vect = gu.examples.get_path("everest_rgi_outlines")
+
+# Open files
+rast = gu.Raster(filename_rast)
+vect = gu.Vector(filename_vect)
+```
+
+A {class}`~geoutils.Raster` is a composition class with four main attributes: a {class}`numpy.ma.MaskedArray` as {attr}`~geoutils.Raster.data`, a
+{class}`pyproj.crs.CRS` as {attr}`~geoutils.Raster.crs`, an {class}`affine.Affine` as {attr}`~geoutils.Raster.transform`, and a {class}`float` or
+{class}`int` as {attr}`~geoutils.Raster.nodata`.
+
+
+```{code-cell} ipython3
+# The opened raster
+rast
+```
+
+```{important}
+When a file exists on disk, {class}`~geoutils.Raster` is linked to a {class}`rasterio.io.DatasetReader` object for loading the metadata. The array will be
+**loaded in-memory implicitly** when {attr}`~geoutils.Raster.data` is required by an operation.
+
+See {ref}`core-lazy-load` for more details.
+```
+
+A {class}`~geoutils.Vector` is a composition class with a single main attribute: a {class}`~geopandas.GeoDataFrame` as {attr}`~geoutils.Vector.ds`, for which
+most methods are wrapped directly into {class}`~geoutils.Vector`.
+
+```{code-cell} ipython3
+:tags: [hide-output]
```

Comment:
I would not hide the output, as one may wonder why there is no output for the Vector case...

Reply:
Done